pkgs/development/interpreters/python/default.nix

```diff
@@ -80,7 +78,7 @@ with pkgs;
           optionalExtensions = cond: as: if cond then as else [];
           python2Extension = import ../../../top-level/python2-packages.nix;
           extensions = lib.composeManyExtensions ((optionalExtensions (!self.isPy3k) [python2Extension]) ++ pkgs.pythonPackagesExtensions ++ [ overrides ]);
-          aliases = self: super: lib.optionalAttrs config.allowAliases (import ../../../top-level/python-aliases.nix lib self super);
+          aliases = self: super: lib.optionalAttrs pkgs.config.allowAliases (import ../../../top-level/python-aliases.nix lib self super);
```

Comment:
Shouldn't that be just config as an input?

Reply:
all pkgs removed